apps/vast/vast-bridge.py

```diff
@@ -126,7 +126,9 @@ async def receive(pub_endpoint: str, topic: str, intel_queue: asyncio.Queue):
     poller.register(socket, zmq.POLLIN)
     logger.info(f"Receiving via ZMQ on topic {pub_endpoint}/{topic}")
     while True:
-        socks = dict(poller.poll(timeout=100))
+        socks = dict(
+            poller.poll(timeout=100)
+        )  # note that smaller timeouts may increase CPU load
```

Comment:
This comment is better, but I still don't understand how the knob operates. What are the two ends of the spectrum? Latency and throughput, presumably, but what that means in the context of TB would be the insight needed here.

Reply:
This "knob" is not exposed because it should not be tuned. Given the surrounding loop, it becomes quite clear that this is a busy polling part and the busier one polls, the more load it will cause.